ivy/functional/frontends/numpy/manipulation_routines/changing_number_of_dimensions.py

```diff
@@ -0,0 +1,15 @@
+# global
+import ivy
+
+
+# squeeze
+def squeeze(
+    x,
+    axis,
+    /,
+):
+    ret = ivy.squeeze(x, axis)
+    return ret
```

Comment:
one last thing, the variable is not needed and can directly be `return ivy.squeeze...`